pkgs/applications/misc/qtpass/default.nix

```diff
@@ -1,20 +1,29 @@
-{ stdenv, mkDerivation, fetchFromGitHub, git, gnupg, pass, qtbase, qtsvg, qttools, qmake, makeWrapper }:
+{ stdenv, mkDerivation, fetchFromGitHub, fetchpatch
+, git, gnupg, pass, qtbase, qtsvg, qttools, qmake, makeWrapper
+}:
 
 mkDerivation rec {
   pname = "qtpass";
-  version = "1.2.3";
+  version = "1.3.0";
 
   src = fetchFromGitHub {
     owner  = "IJHack";
     repo   = "QtPass";
     rev    = "v${version}";
-    sha256 = "1vfhfyccrxq9snyvayqfzm5rqik8ny2gysyv7nipc91kvhq3bhky";
+    sha256 = "0v3ca4fdjk6l24vc9wlc0i7r6fdj85kjmnb7jvicd3f8xi9mvhnv";
   };
 
   buildInputs = [ git gnupg pass qtbase qtsvg qttools ];
 
   nativeBuildInputs = [ makeWrapper qmake ];
 
+  patches = [
+    (fetchpatch {  # Fix missing app icon on Wayland
```

Comment:
If this commit is already in master, please put a comment about which version will no longer require the patch.

Reply:
Done. I was relying on the fact that once the next version is released the patch will fail to apply. No harm in documenting it though.